### PR TITLE
Support symptom date and test date

### PR DESCRIPTION
**Pull request by sethvargo**
Fixes GH-224

**Release Note**

```release-note
Support symptom date and test date in the UI
```

### CODE REVIEW

File: pkg/controller/home/home.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func (c *Controller) HandleHome() http.Handler {
 		m := controller.TemplateMapFromContext(ctx)
 		// Set test date params
 		now := time.Now().UTC()
+		m["todayDate"] = now.Format("2006-01-02")
```
Comment (mikehelmick) [inline pkg/controller/home/home.go:77]: do we need both todaysDate and maxDate?

File: pkg/controller/issueapi/issue.go [hunk 2]
```diff
@@ -30,15 +30,34 @@ import (
 func (c *Controller) HandleIssue() http.Handler {
 	logger := c.logger.Named("issueapi.HandleIssue")
 
+	type FormData struct {
```
Comment (mikehelmick) [inline pkg/controller/issueapi/issue.go:33]: why this change?

I'm concerned about drift in API definition.

Comment (mikehelmick) [reply pkg/controller/issueapi/issue.go:33]: test date should be added to the adminapi version as well

Comment (sethvargo) [reply pkg/controller/issueapi/issue.go]: I think I addressed this in 237483a

**Comment by sethvargo** [comment]
@mikehelmick fixed
